src/pages/VolunteersJobListingFS.page

```diff
@@ -159,6 +168,7 @@
                     	if (x == '') x = JobShiftDateTimeString('{!shift.Start_Date_Time__c}', '{!shift.Duration__c}');
                         OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '{!shift.Id}', '{!JSENCODE(job.Name)}', x); 
                         return false;"
+                        style="{!IF(job.External_Signup_Url__c == null, '', 'display: none;')}"
```

Comment:
@davidhabib I wanted to make sure you received my note - I made the update you requested yesterday, and I wanted to check if there is anything else in this pull request that you would like me to change. Thank you.